Services/Dashboard/ItemsBlock/classes/class.ilPDSelectedItemsBlockSelectedItemsViewGUI.php

```diff
@@ -1,24 +1,26 @@
 <?php
```

Comment:
Thx for the refactoring. I would highly appreciate if you could also add strcit type here.

Reply:
Removed strict type because the code is not type strict.